lib/datapacktypes/omniscript.js

```diff
@@ -10,24 +10,37 @@ var OMNI_BULK_RECORDS_LIMIT_MIN = 1000;
 OmniScript.prototype.createTitleObjects = function(input) {
     var dataPackData = input.dataPackData;
     var titleObjects = [];
-
-    if (dataPackData.VlocityDataPackData['%vlocity_namespace%__OmniScript__c'][0]['%vlocity_namespace%__Element__c']) {
-        dataPackData.VlocityDataPackData['%vlocity_namespace%__OmniScript__c'][0]['%vlocity_namespace%__Element__c'] = this.vlocity.datapacksexpand.sortList(dataPackData.VlocityDataPackData['%vlocity_namespace%__OmniScript__c'][0]['%vlocity_namespace%__Element__c'], '%vlocity_namespace%__OmniScript__c');
+    let omniScriptDataField = '%vlocity_namespace%__OmniScript__c',
+        element_c_DataField = '%vlocity_namespace%__Element__c';
+    if(!dataPackData.VlocityDataPackData['%vlocity_namespace%__OmniScript__c']){
+        omniScriptDataField = 'OmniScript__c';
```

Comment:
You can just do this one check then set all the fields to not use the %vlocity_namespace%. If 1 doesn't have namespace then none of them have it.

Reply:
Or you could just use the ${namespace} syntax in this file.

Reply:
I was considering that primary and secondary keys and their functionality may or may not be same in case of namespace.